client/src/pages/signin/Signin.js

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from "react"
+import './Signin.scss'
```

Comment:
can we use the same styling flow as our example - we rather use what material-ui uses - but I would confirm with the result of your team 
See this for more details - https://material-ui.com/styles/basics/

If you are not using Hooks and functional components, I generally like the Higher-order component API - this will allow you to use the theme we created for you, as opposed to just using SASS

Reply:
@skassam21 - Just to clarify a couple of things along those lines. I have a few queries:
1. If we are using material-ui-styles would all the styles have to be based of just material ui?
2. Can you please share details of the theme that you are referring to and where i can reference it from?
3. I have used some customized buttons in the .scss file, can i retain its usage?
4. I have created a customized layout of the UI page which mimics the wireframes. It also has background images as scss styles. If this approach is not suitable can you please share suggestions on how it can be done with material ui?

Reply:
I would rather not use scss files tbh and move the css into how material-ui does styles (so probably using higher-order component API). Everything you are doing in scss can be done using material-ui-styles 

I'm okay with merging this though if this adds a lot of extra time given that we want something to show for the first demo

Reply:
The styling has been updated to the material UI styling format as per the suggestions
- removed the scss styling and using the higher order component API